mdtraj/formats/mol2.py

```diff
@@ -99,7 +99,23 @@ def load_mol2(filename):
     # If this is a sybyl mol2, there should be NAN (null) values
     if atoms_mdtraj.element.isnull().any():
         # If this is a sybyl mol2, I think this works generally.
-        atoms_mdtraj["element"] = atoms.atype.apply(lambda x: x.split(".")[0])
+        # Argument x is being passed as a list with only one element.
+        def to_element(x):
+            if isinstance(x, (list, tuple)):
+                assert len(x) == 1
+                x = x[0]
+
+            if '.' in x:  # orbital-hybridizations in SYBL
+                return x.split('.')[0]
```

Comment:
Since the values after a `.` are trashed, does this mean I can use that part for some flexibility? For example, carrying atom types like `C.opls_135` should be parsed to an elemental carbon

Reply:
Precisely!